app/src/main/java/com/itsaky/androidide/activities/editor/EditorHandlerActivity.kt

```diff
@@ -195,6 +220,26 @@ open class EditorHandlerActivity : ProjectHandlerActivity(), IEditorHandler {
     openFile(File(cache.selectedFile))
   }
 
+  private fun onReadOrUpdateEditorWorkspaceSettings(workspaceSettings: EditorWorkspaceSettings?) {
+    workspaceSettings ?: return
+    completionsMatchLower = workspaceSettings.completionsMatchLower
+    drawLeadingWs = workspaceSettings.drawLeadingWs
+    drawTrailingWs = workspaceSettings.drawTrailingWs
+    drawInnerWs = workspaceSettings.drawInnerWs
+    drawEmptyLineWs = workspaceSettings.drawEmptyLineWs
+    drawLineBreak = workspaceSettings.drawLineBreak
+    fontSize = workspaceSettings.fontSize
+    tabSize = workspaceSettings.tabSize
+    autoSave = workspaceSettings.autoSave
+    fontLigatures = workspaceSettings.fontLigatures
+    visiblePasswordFlag = workspaceSettings.visiblePasswordFlag
+    wordwrap = workspaceSettings.wordwrap
+    useSoftTab = workspaceSettings.useSoftTab
+    deleteTabsOnBackspace = workspaceSettings.deleteTabsOnBackspace
+    stickyScrollEnabled = workspaceSettings.stickyScrollEnabled
+    pinLineNumbers = workspaceSettings.pinLineNumbers
```

Comment:
Other project will be affected only once and next time that project also has its own settings.json file then it will be applied while opening the project.

Reply:
It must not affect anything other than the preferences for that specific project.

Reply:
Yeah